I realize, this is build time script.  So it should use in
source dictionary data via ../anthy-test-conf. 

dic is set from command line and it is OK as is.

diff --git a/calctrans/proccorpus.c b/calctrans/proccorpus.c
index ec00787..8848700 100644
--- a/calctrans/proccorpus.c
+++ b/calctrans/proccorpus.c
@@ -163,7 +163,7 @@ main(int argc, char **argv)
     else
       read_db(db, argv[i]);
 
-  anthy_conf_override("CONFFILE", "../anthy-conf");
+  anthy_conf_override("CONFFILE", "../anthy-test-conf");
   anthy_conf_override("DIC_FILE", dic);
   anthy_init();
   anthy_set_personality("");
